app/controllers/general.php

```diff
@@ -173,6 +173,10 @@
      */
     if (App::getEnv('_APP_OPTIONS_FORCE_HTTPS', 'disabled') === 'enabled') { // Force HTTPS
         if ($request->getProtocol() !== 'https') {
+            if($request->getMethod() !== Request::METHOD_GET) {
+                throw new Exception('Method unsupported over HTTP.', 405, Exception::GENERAL_METHOD_UNSUPPORTED);
```

Comment:
Here it says `405` - but in the errors its a `500`?

Reply:
Changed to 405 everywhere.